src/languages/en.ts

```diff
@@ -589,6 +589,9 @@ export default {
             invalidSplit: 'Split amounts do not equal total amount',
             other: 'Unexpected error, please try again later',
             genericCreateFailureMessage: 'Unexpected error requesting money, please try again later',
+            receiptFailureMessage: "The receipt didn't upload. ",
+            saveFileMessage: 'Save the file ',
```

Comment:
NAB - When first looking at it, I wasn't sure if "Save the file" meant "Downloading" it, or "Retrying to save it Expensify side" - Maybe renaming it it "Download the file" would remove this confusion?